tensorflow_cloud/run.py

```diff
@@ -93,6 +96,15 @@ def run(entry_point=None,
             Command line arguments to pass to the `entry_point` program.
         stream_logs: Boolean flag which when enabled streams logs back from
             the cloud job.
+        cloud_bucket_name: Optional string that specifies the cloud storage
```

Comment:
This parameter is specific to containerization, so that should be reflected in the name.

I'd like to avoid confusion between concepts like "cloud storage location to use in place of the local disk for things like checkpoints / TensorBoard / input data" and "where to store the docker image when using Cloud Build"

Reply:
Renamed to `docker_image_bucket_name`.